src/launcher/src/main/resources/database/projections.xml

```diff
@@ -60,18 +60,5 @@
                                  constraintName="FK_users_roles_users_id"
                                  referencedTableName="users"
                                  referencedColumnNames="id"/>
-
```

Comment:
The 'encodedpassword' column needs to be removed from the users table.

Reply:
Resolved in [Commit-fcb5e4](https://github.com/everest-engineering/lhotse/commit/fcb5e4fec51b77dbc475dedeb38c9d742f4602b2)